### ISSUE TITLE
Authentication needed

### ISSUE THREAD

**Issue by ngbrown** [dakrone/elasticsearch-in-action#8]
I've installed Elastic Search 5 with the [steps here](https://www.elastic.co/start) and when I try to run the `./populate.sh` script, I get authentication errors:

```
Creating 'get-together' index...
{"error":{"root_cause":[{"type":"security_exception","reason":"missing authentication token for REST request [/get-together]","header":{"WWW-Authenticate":"Basic realm=\"security\" charset=\"UTF-8\""}}],"type":"security_exception","reason":"missing authentication token for REST request [/get-together]","header":{"WWW-Authenticate":"Basic realm=\"security\" charset=\"UTF-8\""}},"status":401}
```

It seems that installing X-Pack turned on authentication which needs [turned back off](https://www.elastic.co/guide/en/x-pack/current/security-settings.html#general-security-settings).

Of course, then I had to use `./populate.5x.sh` from the still open pull request #7.

**Comment by dakrone** [comment]
Yes, there are currently no steps documented for using the script with x-pack. That's something that can certainly be added though, I'll work on this once #7 is merged unless someone beats me to it.
